frontend/src/component/Account/UserSpacesConfigLine.styl

```
@@ -0,0 +1,8 @@
+@import "~tracim_frontend_lib/src/css/Variable.styl"
+
+@media (max-width: max-sm)
+  .spaceconfig__table__leave_space_cell
+    & .iconbutton__label
```

```suggestion
    & .iconbutton__text
```
As the property is named `text` in `IconButton`